src/Templates/src/templates/maui-blazor/.template.config/ide.host.json

```diff
@@ -4,7 +4,15 @@
     "unsupportedHosts": [
         {
             "id": "vs",
-            "version": "(,17.1)"
+            "version": "(,17.2)"
```

Comment:
unsupportedHosts ? Shouldn0t it be supportedHosts?

Reply:
No I think the intent is to say the unsupported versions are 17.2 or older. This looks correct to me.

Reply:
Though this probably needs to wait until 17.3p1

Reply:
yeah the terms are a bit confusing. The way I changed it now it's unavailable for versions earlier than 17.2. Or; everything from 17.2 and up is supported

Reply:
This is a NugetVersion range.  ) indicates non-exclusive.

The way it was before the change would result in the template being "unsupported" on all versions less than 17.1, but shown on 17.1 or later. The new change would result in the template not being shown on 17.1 or older, but being shown on 17.2 or later.

The reason it was designed this way rather then supportedHosts was because in general, templates are generally widely supported regardless of the host or host version. If we had gone the route of "supportedHosts", if you had a template that went from being supported on all hosts vs supported on all host EXCEPT a range of a particular host, the author would need to know to know the host id for each host and add a section for each.  This model allows a more explicit opt out of known versions a template does not want to support.